packages/bot/src/stores/journal-store.js

```diff
@@ -28,17 +28,14 @@ export default class JournalStore {
         this.pushMessage(data, message_types.success);
     }
 
-    @action.bound
-    onLogError(data) {
-        if (unrecoverable_errors.some(x=>x.name && x.name === data.name)) {
-            this.root_store.run_panel.reset();
-        }
-        this.pushMessage(data, message_types.error);
-    }
-
     @action.bound
     onError(data) {
-        this.root_store.run_panel.reset();
+        if (unrecoverable_errors.some(x=>x && x === data.name)) {
```

Comment:
You can use `includes` instead as it's a lot clearer imo. 

```suggestion
        if (unrecoverable_errors.includes(data.name) {
```